application/helpers/frontend_helper.php

```diff
@@ -1341,13 +1341,14 @@ function renderRenderWayForm($renderWay, array $scenarios, $sTemplateViewPath, $
 }
 
 /**
- * Resets all session variables for this survey
+ * Resets all needed session variables for this survey,
+ * keep related current survey : s_lang, srid, token …
+ * @see killSurveySession to reset ALL session
  * @param int $surveyid
  * @return void
  */
 function resetAllSessionVariables($surveyid)
 {
-    Yii:app()->session->regenerateID(true);
```

Comment:
Hm, what's the commit message for this line? Before it's removed here, I mean.

Reply:
https://github.com/LimeSurvey/LimeSurvey/commit/47107ee0e033727a009d2e696fe204bde374b996

 Fixed issue: [security] Possible session fixation on survey entry with token

Reply:
PS : see the comment here : i put the line of the commit in comment …